### PR TITLE
 fix a typo

### PR DESCRIPTION
**Pull request by lijianfeng1993**

### CODE REVIEW

File: contributors/design-proposals/api-machinery/aggregated-api-servers.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ There were other alternatives that we had discussed.
 
 * Validate servers: We should have some conformance tests that validate that the
   servers follow kubernetes api-conventions.
-* Provide centralised auth service: It is very hard to ensure homogeneous auth
+* Provide centralized auth service: It is very hard to ensure homogeneous auth
```
Comment (deads2k) [inline contributors/design-proposals/api-machinery/aggregated-api-servers.md]: heh.  Anti-british?

**Comment by deads2k** [comment]
@lavalamp @ericchiang @liggitt do we have something saying we use standard american spellings instead of british ones?  

I have no real preference on the first change.  I'll wait a day for comments before tagging.

File: contributors/design-proposals/api-machinery/aggregated-api-servers.md (hunk 1 above)
Comment (cblecker) [reply contributors/design-proposals/api-machinery/aggregated-api-servers.md]: Yeah, the UK spelling is fine. We can remove this change :)

Comment (lijianfeng1993) [reply contributors/design-proposals/api-machinery/aggregated-api-servers.md]: thanks a lot.

**Pull request merged**